client/src/components/NewReviewForm.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react"
+
+const NewReviewForm = ({ postNewReview, currentUser }) => {
```

Comment:
the front end form should only be responsible for information from the user. the back-end express is keeping track of the currently signed in user through the session. when a new review is to be added, we can access `req.user` in the server endpoint to get the id of the current user